### PR TITLE
IA-1406 fix google sign in discrepancy

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-1406

### CODE REVIEW

File: terra-jupyter-python/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.0.3 - 10/18/2019
+
+- Update `terra-jupyter-base` version to `0.0.4`
+`Image URL: us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.4`
```
Comment (akarukappadath) [inline terra-jupyter-python/CHANGELOG.md]: this should be `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.3` right?

File: terra-jupyter-base/scripts/kernel/kernel_bootstrap.sh [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
-#   ~jupyter-user/<workspace-name>/notebook.ipynb
+#   ~jupyter-user/<workspace-name>/<mode>/notebook.ipynb
 #
 # It exploits the fact that the CWD of a launching notebook is named after the workspace.
 # If notebooks are ever launched from other directories, this script will break.
 
 # The workspace name is simply the CWD of the running notebook.
 PWD="$(pwd)"
-export WORKSPACE_NAME="$(basename "$PWD")"
+export WORKSPACE_NAME=`basename $(dirname $(pwd))`
```
Comment (Qi77Qi) [inline terra-jupyter-base/scripts/kernel/kernel_bootstrap.sh]: this fixes the `WORKSPACE_NAME` issue after welder rollout

**Comment by Qi77Qi** [comment]
@rtitle , we'd have to update it manually. https://broadworkbench.atlassian.net/browse/IA-1446 will fix the problem

**Comment by melissachang** [comment]
Hi, I'm running into WORKSPACE_NAME bug on app.terra.bio. When will this fix be on app.terra.bio?

**Comment by rtitle** [comment]
Hi @melissachang ,

We're targeting the fix for Monday's release (10/28). Once released you'll need to stop/start your runtime to get the fix.

In the meantime, this is a workaround to get the workspace name in a python variable:

```
import os
workspace = os.path.basename(os.path.dirname(os.getcwd()))
```

File: terra-jupyter-base/scripts/kernel/kernel_bootstrap.sh (hunk 2 above)
Comment (mbookman) [reply terra-jupyter-base/scripts/kernel/kernel_bootstrap.sh]: Could the comment in the code here be updated? It is no longer accurate:

```
# The workspace name is simply the CWD of the running notebook.
```

Should be something like:

```
# Notebooks are copied into a subdirectory, such as "edit" or "safe", which is the CWD.
# The parent directory is the name of the workspace.
```

Thanks!

**Pull request merged**